iothub/device/src/CommandResponse.cs

```diff
@@ -40,8 +33,11 @@ public CommandResponse(int status)
         }
 
         /// <summary>
-        /// The command response status code indicating success or failure.
+        /// The command response status code indicating success or failure. This is usually an HTTP Status Code e.g. 200, 400.
```

Comment:
I'd suggest putting this <remarks>.

Also, perhaps refer to CommonResponseCodes type.